radixdlt/babylon-wallet-android
# [ABW-3974] - Shield Setup - Regular access

[ABW-3974](https://radixdlt.atlassian.net/browse/ABW-3974)

## Description
This PR introduces the [Setup Regular Access](https://zpl.io/EdmE5ng) flow. It also defines a  wrapper around Sargon's `SecurityShieldBuilder` operations, while holding its internal state and making it accessible wherever required during the Security Shield building flow.


## How to test

1. Make sure you have at least enough sources added (you can use the profile provided [here](https://rdxworks.slack.com/archives/C031A0V1A1W/p1734599283252709))
2. Go to Security Center -> Security Shields -> Next -> Next -> Start
3. Select some factors
4. Play around with different factors selection and navigate back & forth to see how the builder state is being preserved

[Video](https://rdxworks.slack.com/archives/C03Q8QK1GLW/p1735905124259389?thread_ts=1735905056.367019&cid=C03Q8QK1GLW)

[ABW-3974]: https://radixdlt.atlassian.net/browse/ABW-3974?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
